code/testapp/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+}
+
+android {
+      namespace = "com.adobe.placestestapp"
```

Comment:
According to the wiki, "Refactor Test application to separate package: If your project's test application doesn't have a unique namespace, move it to a namespace like com.adobe.marketing.mobile.<extension>.testapp for clearer separation. Make sure the app name follows similar naming (ex UserProfile Test App)."

Not sure if you want to refactor to 
com.adobe.marketing.mobile.places.testapp

Reply:
I will refactor this in my next PR